src/output.ts

```diff
@@ -61,3 +73,12 @@ function extractErrorMessage(error: unknown) {
 
   return 'Unknown error';
 }
+
+function formatStats(stats: ResponseStats) {
+  const parts = [
+    `time: ${(stats.responseTime / 1000).toFixed(1)} s`,
+    `tokens: ${stats.prompt_tokens} in + ${stats.completion_tokens} out`,
```

Comment:
maybe we could also add a sum of that? QoL improvement to avoid checking stats with calc opened :P

Reply:
Why would you like to have a sum? input and output tokes have differnt price, etc